Lib/test/test_genericalias.py

```diff
@@ -302,6 +301,33 @@ def test_weakref(self):
                 alias = t[int]
                 self.assertEqual(ref(alias)(), alias)
 
+    def test_abc_callable(self):
```

Comment:
Can you also add a test that e.g. `F = Callable[[int, T], T]` followed by `F[str]` indeed returns `Callable[[int, str], str]`?